radixdlt-java/src/main/java/com/radixdlt/client/application/RadixApplicationAPI.java

```diff
@@ -825,6 +768,62 @@ public UnsignedAtom buildAtomWithFee(List<ParticleGroup> particleGroups) {
 		return new UnsignedAtom(new Atom(allParticleGroups, metaData));
 	}
 
+	/**
+	 * Represents an atomic transaction to be committed to the ledger
+	 */
+	public final class Transaction {
```

Comment:
This one can't be static because of dependency on `buildDisconnectedResult(Action[])`, right?